src/main/java/org/fundacion/jala/converter/core/facade/MetadataFacade.java

```diff
@@ -28,7 +30,7 @@
      * @param outputFileName a String with the new file's name.
      * @param fileStorageService an object to create the path.
      */
-    public static void extractMetadata(final boolean metadata, final String outputFileName, final FileStorageService fileStorageService) {
+    public static void extractMetadata(final boolean metadata, final String outputFileName, final FileStorageService fileStorageService) throws MetadataException {
```

Comment:
Jessi, Is this line not longer than 120 characters?

Reply:
Thanks, I have just fixed it